package.json

```diff
@@ -19,10 +19,11 @@
     "qunitjs": "^2.3.2"
   },
   "scripts": {
-    "start": "live-server --port=8000 --open=./examples",
-    "test": "istanbul cover test/test.js",
+    "check-coverage": "istanbul cover test/test.js && istanbul check-coverage --statements 100 --functions 100 --lines 100 --branches 100",
+    "postinstall": "npm test && npm start",
```

Comment:
need to _delete_ this line as it causes the build to "hang" (_unsurprisingly, in retrospect! doh!_)
https://travis-ci.org/dwyl/learn-elm-architecture-in-javascript/builds/394115145#L551